resources/html/test/testPlayground.html

```diff
@@ -0,0 +1,188 @@
+<!--
+    This file is a html page with various element used for testing candybean's
+    functionalities
+-->
+
+<html>
+    <head>
+        <title>Candybean Test Page</title>
+        <script type="text/javascript">
+/*
+ * Toggles the visibility of the paragraph with id 'writing', can be used
+ * for testing candybean functions which require and element to variably
+ * be either visible or invisible or on/off the DOM.
+ */
+function toggleDOMWriting() {
+    var para = document.getElementById('writing');
+    var parent = document.getElementById('writingDiv');
+    if (para != null) {
+        parent.removeChild(para);
+    }
+    else{
+        var para = document.createElement('span');
+        para.setAttribute('id', 'writing');
+        var text = document.createTextNode('Click button to hide me');
+        para.appendChild(text);
+        parent.appendChild(para);
+    }
+}
+
+/*
+ * Toggles the visibility of element
+ *
+ * String -> _
+ * @para element The id of the element to toggle
+ */
+function toggleVisibility(element) {
+    var elem = document.getElementById(element);
+
+    if (elem.className == 'hidden') {
+        elem.className = 'normal'
+    }
+    else {
+        elem.className = 'hidden';
+    }
+}
+        </script>
+    </head>
+
+    <style>
+.onScreen {
+    position: absolute; left: 50; top:50;
+}
+.hidden {
+    visibility:hidden;
+}
```

Comment:
Uhh didn't refresh my page when you commented on this. I noticed it after and deleted my previous comment